acceptance-tests/proxy_protocol_test.go

```diff
@@ -23,44 +25,88 @@ var _ = Describe("Proxy Protocol", func() {
-			haproxyBackendPort:    haproxyBackendPort,
-			haproxyBackendServers: []string{"127.0.0.1"},
-			deploymentName:        deploymentNameForTestNode(),
-		}, []string{opsfileProxyProtocol}, map[string]interface{}{}, false)
-
-		closeLocalServer, localPort := startDefaultTestServer()
-		defer closeLocalServer()
-
-		closeTunnel := setupTunnelFromHaproxyToTestServer(haproxyInfo, haproxyBackendPort, localPort)
-		defer closeTunnel()
-
-		By("Waiting for monit to report that HAProxy is healthy")
-		// Since the backend is now listening, HAProxy healthcheck should start returning healthy
-		// and monit should in turn start reporting a healthy process
-		// We will wait up to one minute for the status to stabilise
-		Eventually(func() string {
-			return boshInstances(deploymentNameForTestNode())[0].ProcessState
-		}, time.Minute, time.Second).Should(Equal("running"))
-
-		By("Sending a request with Proxy Protocol Header to HAProxy traffic port")
-		err := performProxyProtocolRequest(haproxyInfo.PublicIP, 80, "/")
-		Expect(err).NotTo(HaveOccurred())
-
-		By("Sending a request without Proxy Protocol Header to HAProxy")
-		_, err = http.Get(fmt.Sprintf("http://%s", haproxyInfo.PublicIP))
-		expectConnectionResetErr(err)
-
-		By("Sending a request with Proxy Protocol Header to HAProxy health check port")
-		err = performProxyProtocolRequest(haproxyInfo.PublicIP, 8080, "/health")
-		Expect(err).NotTo(HaveOccurred())
-
-		By("Sending a request without Proxy Protocol Header to HAProxy health check port")
-		_, err = http.Get(fmt.Sprintf("http://%s:8080/health", haproxyInfo.PublicIP))
-		expectConnectionResetErr(err)
+	Context("expect_proxy", func() {
+		opsfileExpectProxyProtocol := `---
+# Enable Proxy Protocol
+- type: replace
+  path: /instance_groups/name=haproxy/jobs/name=haproxy/properties/ha_proxy/accept_proxy?
+  value: false
+- type: replace
+  path: /instance_groups/name=haproxy/jobs/name=haproxy/properties/ha_proxy/expect_proxy?
+  value: 
+  - 10.0.0.0/8 # Bosh Network CIDR
+`
+		It("Correctly proxies request with Expect Proxy CIDRs list", func() {
+			haproxyBackendPort := 12000
+			haproxyInfo, _ := deployHAProxy(baseManifestVars{
+				haproxyBackendPort:    haproxyBackendPort,
+				haproxyBackendServers: []string{"127.0.0.1"},
+				deploymentName:        deploymentNameForTestNode(),
+			}, []string{opsfileExpectProxyProtocol}, map[string]interface{}{}, true)
+
+			closeLocalServer, localPort := startDefaultTestServer()
+			defer closeLocalServer()
+
+			closeBackendTunnel := setupTunnelFromHaproxyToTestServer(haproxyInfo, haproxyBackendPort, localPort)
+			defer closeBackendTunnel()
+
+			By("Not expecting Proxy Protocol for requests from IPs not in the list")
+			_, err := http.Get(fmt.Sprintf("http://%s", haproxyInfo.PublicIP))
+			expectConnectionResetErr(err)
+
+			err = performProxyProtocolRequest(haproxyInfo.PublicIP, 80, "/")
+			Expect(err).NotTo(HaveOccurred())
+
+			By("Expecting Proxy Protocol for requests from IPs in the list")
```

Comment:
I cannot find the request and the "Expect" check for this test case.

Reply:
see my comment below